Resources/Prototypes/Reagents/medicine.yml

```diff
@@ -203,7 +203,7 @@
   color: "#32cd32"
   worksOnTheDead: true # Frontier
   metabolisms:
-    Medicine:
```

Comment:
i have concerns about removing the medicine property, instead add a type to the medicine to flag it as cryogenics this way we do not inadvertently lose a medicine mechanic

Reply:
We aren't losing mechanics.  Per the definition of ReagentPrototype, the `metabolisms` field is as follows:
```
        [DataField(serverOnly: true)]
        public FrozenDictionary<ProtoId<MetabolismGroupPrototype>, ReagentEffectsEntry>? Metabolisms;
```

This groups a set of effects to a metabolic group (used to be `Medicine` here, now `Cryogenic`).
This metabolic group can be specified by organs that can process it.  Every single instance of a medicine-processing organ I found had Cryogenic added to it.

This requires any new species (mostly player species, to be honest) to have consistent organ metabolisms, otherwise they aren't healed by cryogenic meds.  Worst case, a simple bug to fix.